docs/frame/evm.md

```diff
@@ -0,0 +1,45 @@
+# EVM pallet
+
+The EVM module allows unmodified EVM code to be executed in a
+Substrate-based blockchain.
+
+## EVM engine
+
+The EVM module uses
+[`SputnikVM`](https://github.com/rust-blockchain/evm) as the
+underlying EVM engine.
+
+## Execution lifecycle
+
+There are a separate set of accounts managed by the EVM
+module. Substrate based accounts can call the EVM Module to deposit or
+withdraw balance from the Substrate base-currency into a different
+balance managed and used by the EVM module. Once a user has populated
+their balance, they can create and call smart contracts using this
+module.
+
+There's one-to-one mapping from Substrate accounts and EVM external
+accounts that is defined by a conversion function.
```

Comment:
Isn't it the case that Substrate (with default AccountId32) has a larger account space than Ethereum? Thus the mapping can't truly be 1:1.

Reply:
Fixed! Changed this to:

> Substrate accounts and EVM external accounts are mapped via customizable conversion functions.